scripts/sst_garbd_test.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
This file is used for testing purposes only. Wouldn't it be better to keep it in mysql-test/std_data/?

Reply:
Good idea, moved.